tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -2741,84 +2668,45 @@ namespace OutputProcessor {
                                             "1"};
         EXPECT_EQ(errorData0, errorData[0]);
 
-        ASSERT_EQ(2, state->dataOutputProcessor->NumEnergyMeters);
-        ASSERT_EQ(2ul, state->dataOutputProcessor->EnergyMeters.size());
+        ASSERT_EQ(2ul, op->meters.size());
     }
 
+    // We're not really doing things this way anymore
     TEST_F(SQLiteFixture, OutputProcessor_validateNStandardizeMeterTitles)
     {
-        std::vector<std::vector<std::string>> input_map = {{"J", "ELECTRICITY", "INTERIOR LIGHTS", "endUseSub", "HVAC"},
-                                                           {"J", "ELECTRICITY", "INTERIOR LIGHTS", "endUseSub", "SYSTEM"},
-                                                           {"J", "ELECTRICITY", "INTERIOR LIGHTS", "endUseSub", "PLANT"},
-                                                           {"J", "ELECTRICITY", "INTERIOR LIGHTS", "endUseSub", "BUILDING", "zoneName"},
+        auto &sql = state->dataSQLiteProcedures->sqlite;
+        std::vector<std::vector<std::string>> input_map = {{"J", "ELECTRICITY", "INTERIORLIGHTS", "endUseSub", "HVAC"},
```

Comment:
No need to test different string variations and misspellings snce these are all enums now.